components/SearchResultsList.vue

```diff
@@ -10,24 +10,35 @@
                     <use xlink:href="../assets/images/hamburger-list-icon.svg#hamburger-list-icon-svg" />
                 </svg>
-                <span class="text-primary-text/50 pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">{{
-                    $t('searchResultsList.filters') }}</span>
+                <span
+                    class="text-primary-text/50 pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">{{
+                        $t('searchResultsList.filters') }}</span>
             </button>
         </div>
-        <div id="searchResults" class="flex flex-col overflow-y-scroll h-full">
-            <div @click="resultsStore.setActiveSearchResult(searchResult.professional.id)" :key="index"
-                v-for="(searchResult, index) in resultsStore.searchResultsList" class="results-list flex flex-col">
-                <div class="flex-1 h-24 w-6/8 border-b-2 border-primary/20 p-3
-                    hover:border-transparent hover:bg-primary/5 transition-all hover:cursor-pointer">
-                    <SearchResultsListItem
-                        :name="`${searchResult.professional.names[0].firstName} ${searchResult.professional.names[0].lastName}`"
-                        :degrees="searchResult.professional.degrees"
-                        :facility-name="localeStore.locale?.code == Locale.EnUs ? searchResult.facilities[0]?.nameEn : searchResult.facilities[0]?.nameJa"
-                        :specialties="searchResult.professional.specialties"
-                        :spoken-languages="searchResult.professional.spokenLanguages" />
+        <template v-if="resultsStore.searchResultsList.length > 0">
+            <div id="searchResults" class="flex flex-col overflow-y-scroll h-full">
+                <div @click="resultsStore.setActiveSearchResult(searchResult.professional.id)" :key="index"
+                    v-for="(searchResult, index) in resultsStore.searchResultsList" class="results-list flex flex-col">
+                    <div class="flex-1 h-24 w-6/8 border-b-2 border-primary/20 p-3
+                        hover:border-transparent hover:bg-primary/5 transition-all hover:cursor-pointer">
+                        <SearchResultsListItem
+                            :name="`${searchResult.professional.names[0].firstName} ${searchResult.professional.names[0].lastName}`"
+                            :degrees="searchResult.professional.degrees"
+                            :facility-name="localeStore.locale?.code == Locale.EnUs ? searchResult.facilities[0]?.nameEn : searchResult.facilities[0]?.nameJa"
+                            :specialties="searchResult.professional.specialties"
+                            :spoken-languages="searchResult.professional.spokenLanguages" />
+                    </div>
                 </div>
             </div>
-        </div>
+        </template>
+        <template v-else>
```

Comment:
♻️ I'd suggest refactoring out these template tags. They could probably be replaced by divs.

Reply:
Alright, I will take a look